### DISCUSSION TITLE
竖行文本如何标注？

### DISCUSSION THREAD

**Discussion by SUPEROKMAN** [PaddlePaddle/PaddleOCR#12402] [Q&A]
竖行的集装箱号，按顺时针画的矩形框，标注文本没有空格，这样可否？

![未标题-1](https://github.com/PaddlePaddle/PaddleOCR/assets/24601592/b6fe128b-1f5f-49e2-ac47-d2184dac0361)

用PP-OCRv3处理的这两种竖行文本，结果差别很大，新手，还请指点。

![下载](https://github.com/PaddlePaddle/PaddleOCR/assets/24601592/aee65d27-2192-417a-8b7b-4c9ee4edc6ce)

**Comment by yangoono** [comment]
嗨喽，想知道现在怎么样了，我用PP-OCRv5处理竖版文本，效果很差，只能精确识别出横板文本
